### PR TITLE
Added module: "Displaying the Insights status in the web console" (CCXDEV-3375)

### PR DESCRIPTION
**Pull request by mmuehlfeldRH**
This adds a new module about displaying the Insights status in the web console.
This documentation is for a 4.7 feature.

@openshift/team-documentation

### CODE REVIEW

**Comment by ahardin-rh** [comment]
LGTM!

**Comment by ahardin-rh** [comment]
@yapei Can you please provide a QE review? Thank you!

File: modules/displaying-the-insights-status-in-the-web-console.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+[id="displaying-the-insights-status-in-the-web-console.adoc_{context}"]
```
Comment (ahardin-rh) [inline modules/displaying-the-insights-status-in-the-web-console.adoc:5]: @mmuehlfeldRH One thing I missed is that there is a .adoc in the ID, which must be removed. Thanks!

Comment (mmuehlfeldRH) [reply modules/displaying-the-insights-status-in-the-web-console.adoc]: Good catch. This was a copy/paste issue. :-)
It's fixed now.

File: modules/displaying-the-insights-status-in-the-web-console.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+.Prerequisites
+
+* Your cluster is registered in the {cloud-redhat-com}.
+* Remote health reporting is enabled, which is the default.
+* You are logged in to the {product-title} web console.
+
+.Procedure
+
+. Navigate to *Home* -> *Dashboards* in the {product-title} web console.
```
Comment (yapei) [inline modules/displaying-the-insights-status-in-the-web-console.adoc:18]: The navigation should be `Home -> Overview`

Comment (mmuehlfeldRH) [reply modules/displaying-the-insights-status-in-the-web-console.adoc]: Fixed.

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**